js/runner.go

```diff
@@ -173,7 +173,7 @@ func (r *Runner) newVU(samplesOut chan<- stats.SampleContainer) (*VU, error) {
 	vu := &VU{
 		BundleInstance: *bi,
 		Runner:         r,
-		HTTPTransport:  netext.NewHTTPTransport(transport),
+		Transport:      transport,
```

Comment:
As mentioned in the other comment, it seems like that for NTLM auth caching to still work across requests, we should have some per-VU or per-iteration shared cache of it.

It shouldn't necessarily be the old `netext.NewHTTPTransport()` wrapper that was used for every request, whether it used NTLM auth or not, since most request won't... But we can probably add the NTLM cache map in the `state` (not sure if it should be reset on every iteration, maybe yes unless `noCookiesReset` is enabled?) and use it in the per-request wrapper if NTLM authentication is specified for that request (see other comments about that).